godtossing/Pos/Ssc/GodTossing/Toss/Base.hs

```diff
@@ -136,6 +139,58 @@ computeParticipants (HS.toMap -> richmen) = flip HM.intersection richmen
 sharesDistrInaccuracy :: Fractional a => a
 sharesDistrInaccuracy = 0.05
 
+-- | Internal type for work with Coin.
+type CoinUnsafe = Int
```

Comment:
don't give a fuck, `Int` fits into 2^64 on my machine :peka:
but ok, I will think of it